doc/whats_new.rst

```diff
@@ -1,3 +1,32 @@
+0.3.1
+=====
+
+Changelog
+---------
+
+API changes summary
+...................
+
+    - The function `nilearn.connectome.sym_to_vec` is is deprecated and will be removed in future
+      release. Use :func:`nilearn.connectome.sym_matrix_to_vec` instead.
+
+New features
+............
+
+    - New function :func:`nilearn.connectome.vec_to_sym_matrix` reshapes
+      vectors to symmetric matrices. It acts as the reverse of function
+      :func:`nilearn.connectome.sym_matrix_to_vec`.
+
+    - Add an option allowing to vectorize connectivity matrices returned by the
+      "transform" method of :class:`nilearn.connectome.ConnectivityMeasure`.
+
+    - :class:`nilearn.connectome.ConnectivityMeasure` now exposes an
+      "inverse_transform" method, useful for going back from vectorized
+      connectivity coefficients to connectivity matrices. Also, it allows to
+      recover the covariance matrices for the "tangent" kind.
+
+    - Reworking and renaming of connectivity measures example. 
```

Comment:
Optional: renaming of connectivity measures example to plot_group_level_connectivity ?